projects/news_feed/psl_matches.py

```diff
@@ -52,9 +52,19 @@ def main():
 
     pandas_data_frame = pd.DataFrame(all_fetched_information)
 
-    #printing all fetched info
+    # printing all fetched info
     print(pandas_data_frame.to_string())
 
+    # converting all fetched information into json file format
+    psl_json_data = simplejson.dumps(all_fetched_information)
+
+    # creating a separate json file named as psl_json_data
+    psl_json_data_file = open('E:/TechCrunch/projects/news_feed/data/psl_json_data.txt', 'w')
```

Comment:
also why extension is `.txt` ?

it shall be `.json`

Reply:
txt extension has been replaced by json